docs/metric_deletion_retention.md

```diff
@@ -0,0 +1,49 @@
+Promscale supports series deletion & TimescaleDB supports metric retention. The series deletion is performed on the complete series, whereas the metric retention can be enforced on all the metrics or per metric basis.
+
+## Series Deletion
+
+Promscale exposes an API for deletion of series. This works same as prometheus delete API.
+
+**Note**: The start and end timestamp options are not supported yet. Currently, the `delete_series` API lets you to delete complete series. 
```

Comment:
I do not understand the meaning of **complete series** (opposite, partial series?). It should be **lets you delete the series across the entire time-range.**